core/src/main/resources/db/cgds.sql

```diff
@@ -177,16 +178,34 @@ CREATE TABLE `sample_list_list` (
 );
 
 -- --------------------------------------------------------
+CREATE TABLE `genetic_entity` (
+  `INTERNAL_ID` int(11) NOT NULL AUTO_INCREMENT,
+  `ENTITY_TYPE` varchar(45) NOT NULL,
```

Comment:
Yes, that is correct. But @n1zea144 would also like to have the entity type represented, and I thought it could make sense to have it here. Could also be in a separate table with a foreign key here pointing to it, but that is perhaps not really needed now. This model can easily evolve in subsequent iterations....this would also be my response to the idea of changing this to int now.